openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1500,9 +1500,33 @@ Credential Refresh can be initiated by the Wallet independently from the Credent
 
 It is up to the Credential Issuer whether to update both the signature and the claim values, or only the signature.
 
-# Privacy Considerations
+# Privacy Considerations {#privacy-considerations}
 
-TBD
+Implementers and deployers should utilize mechanisms to protect End-User privacy.
+Means of protecting End-User privacy are described below.
+
+Privacy harms can occur if information about a person is released to another party without the person's consent.
```

Comment:
not only this, multiple seemingly 'innocent' consenting releases can result in privacy erosion through collusion and correlation -- there may be unintended 2nd or 3rd order effects, which is why minimal disclosure is so crucial

Reply:
I added this point to the section.